requirements.txt

```diff
@@ -1,2 +1,2 @@
-psycopg2-binary>=2.7.5
-PyYAML>=3.13
+psycopg2-binary==2.9.3
+PyYAML==6.0
```

Comment:
I wouldn't pin libraries to exact versions unless there are very good reasons for it.

Reply:
Because pulling the code down and running a `pip install -r requirements` with the `>=` deps pulled down versions of libraries that were not compatible with your code. I had to update the code to get it working with the default libraries pulled. This way, over time, the proxy should be guaranteed to work on run, then let the user decide if they want to change the deps.